src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -68,13 +54,13 @@ default String getAsTextHidePrivate() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName());
         if (!getPhone().isPrivate()) {
-            builder.append(" Phone: ").append(getPhone());
+            builder.append(getPhone().getPrintableString());
```

Comment:
I would suggest still using the Formatter.getPrintableString method here, as even though it is only going to process 1 object, it would be consistent with the way you did it earlier.